Opencast-Moodle/moodle-block_opencast
# Admin notification for module cleanup tasks

This PR fixes #192 

### How it works:
Upon cleaning up the modules, there were some conditions that forcefully remove the task not in its normal way, for example when 5 days duration is up or when there is an exception, in thses cases, there were no info regarding to that, but only a mtrace log in the cron texts.
Now, with this PR, those conditions are covered and when they take place Admin will be notified.

### To test this PR:
- Make sure to use the latest moodle opencast versions (v4.0-r1)
- Based on experience (it might not apply everywhere), the new duplicate-event workflow in OC 13 requires more that just seriesId, therefore, it is recommended to test against lower versions (i.e. 11.5), this issue is known and will be resolved soon.
